features/element-hiding.json

```diff
@@ -2304,6 +2304,10 @@
                     {
                         "selector": ".ad",
                         "type": "closest-empty"
+                    },
+                    {
+                        "selector": "div#side_navi > div:has(> div > div.ad)",
+                        "type": "hide"
```

Comment:
I tried "hide-empty" but that didn't work, even after I tried adding the Japanese "adLabelStrings" array.